NEWS

```diff
@@ -1,13 +1,60 @@
-Release a.b
------------
+Release 1.12 (xx February 2021)
+-------------------------------
 
- * Samtools stats now displays the number of supplementary reads in the
+ * samtools stats now displays the number of supplementary reads in the
    SN section. Also, supplementary reads are no longer considered when
    splitting read pairs by orientation (inward, outward, other).
    
- * Samtools stats now counts only the filtered alignments that overlap
+ * samtools stats now counts only the filtered alignments that overlap
    target regions, if any are specified.
 
+ * samtools view now accepts option -N, which takes a file containing
+   read names of interest. This allows the output of only the reads with
+   names contained in the given file. Thanks to Daniel Cameron. (#1324)
+
+ * samtools view -d option now works without a tag associated value, which
+   allows it to output all the reads with the given tag. (#1339; fixed #1317)
+
+ * samtools view -d and -D options now accept integer values associated with
+   tags, not just strings. Thanks to `@dariome` for the suggestion. (#1357)
+
+ * samtools view now works with the filtering expressions introduced by HTSlib.
+   The filtering expression is passed to the program using option -e, e.g.
+   `samtools view -e '!(flag&2048)' align.bam` filters out all the supplementary
+   reads. (#1346)
+
+ * samtools markdup now benefits from an increase in performance in the
+   situation when a single read has tens or hundreds of thousands of duplicates.
+   Thanks to `@denriquez` for reporting the issue. (#1345; fixed #1325)
+
+ * The documentation for samtools ampliconstats has been added to the samtools
+   man page. (#1351)
+
+ * A new FASTA/FASTQ sanitizer script (`fasta-sanitize.pl`) was added, which
+   corrects the illegal characters in the reference names. (#1314) Thanks to
```

Comment:
Suggest `invalid` rather than `illegal` (see [the 4th-last paragraph here](https://www.gnu.org/prep/standards/html_node/GNU-Manuals.html)), and the installation fix is too small to mention IMHO :smile: